Jenkinsfile

```diff
@@ -55,6 +55,7 @@ pipeline {
         dir("${BASE_DIR}"){
           // Skip all the stages except docs for PR's with asciidoc and md changes only
           setEnvVar('ONLY_DOCS', isGitRegionMatch(patterns: [ '.*\\.(asciidoc|md)' ], shouldMatchAll: true).toString())
+          setEnvVar('LICENSE_CHANGES', isGitRegionMatch(patterns: [ '^(go.mod|LICENSE.txt)' ], shouldMatchAll: false).toString())
```

Comment:
This should include `NOTICE.txt` rather than `LICENSE.txt`. But instead of detecting "license changes", I think it would be better to add `make notice` to the Lint branch that doesn't run the whole `make check`:
```
            whenTrue(env.ONLY_DOCS == 'false') {
              cmd(label: "make check-python", script: "make check-python")
              cmd(label: "make check-go", script: "make check-go")
              cmd(label: "make check-go", script: "make notice")
              cmd(label: "Check for changes", script: "make check-no-changes")
            }
```

Reply:
No strong opinion as long as the `make check` is not required sounds fine to me. I'll change it then